CONTRIBUTING.md

```diff
@@ -154,6 +154,7 @@ Make sure that [Travis](http://www.travis-ci.org) greenlights the pull request w
  - use lower case with underscores for method names
  - it is generally preferred to use ASCII operators and identifiers over
    Unicode equivalents whenever possible
+ - function signature should not be followed by an empty line
```

Comment:
Hey @Dawny33! I didn't read your note earlier. Highlighting wasn't added to the other lines, and so I assumed that I shouldn't add it.